src/Components/Artwork/Metadata.tsx

```diff
@@ -80,6 +80,8 @@ const LinkContainer: React.FC<Omit<MetadataProps, "children">> = ({
     return <DisabledLink mt={mt}>{rest.children}</DisabledLink>
   }
 
+  const enablePrefetch = !artwork.sale?.isOpen
```

Comment:
In addition to the auction page, auction lots are displayed on other rails and grids. I think this `Metadata` component is what gets used for rails and grids, right? So other surface areas are also taken care of by this change.

Reply:
You'd think so @starsirius but actually those other rails/grids _will_ prefetch. The reason is b/c those artwork components, have _multiple_ `RouterLink` pointing to the artwork. There's this one in `Metadata`, but the wrapper around `Metadata` - `GridItem` - has one as well - https://github.com/artsy/force/blob/83313ea37b77fb7372201016269cbfefb2450b9b/src/Components/Artwork/GridItem.tsx#L103

So as long as any `RouterLink` to the artwork page is present, the prefetch will happen.

Personally I think that's a bug, why do we have multiple nested links? 

For the prefetching to be skipped for auction works in those other contexts (other rails and grids besides the auction page), you'd need to do a sweep for links to the artwork page in the component and do this same thing, like in `GridItem` at the very least.

Reply:
Got it. Thanks for explaining! Yeah I feel it's tricky to manage all the entry points and I'm curious if it's possible to determine it on the ultimate artwork page, regardless of pre-fetching.